podcast_generator.py

```
@@ -0,0 +1,216 @@
+def limpar_texto(fala):
+    """
+    Removes emotion and tone markers like [rindo], [didática], etc.
+    """
+    return re.sub(r'\[[^\]]*\]', '', fala).strip()
+
+def split_dialogo_by_podcast(full_dialogo):
+    """
+    Splits a large text into individual podcast segments based on titles.
+    """
+    podcasts = []
+    # Matches "Podcast X – Title"
```

Ruff (0.15.18)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)